src/NHSD.BuyingCatalogue.API/NHSD.BuyingCatalogue.API.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" Version="3.1.1" />
     <PackageReference Include="Serilog.AspNetCore" Version="3.2.0" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
+    <PackageReference Include="SpecFlow" Version="3.1.78" />
```

Comment:
Please review this additional package as I am not sure it is needed in the BuyingCatalogue.Api prorject.

Reply:
Removed import and fixed failing test